### ISSUE TITLE
Which Modem/Internet Devices are Supported?

### ISSUE THREAD

**Issue by popeyeotaku** [tschak909/platoterm64#25]
You need to create a list of these somewhere.

**Comment by tschak909** [comment]
Yes. This is planned, I just haven't been able to get to it yet. Perhaps you'd like to help? :)

But to sum it up:

For the Commodore 64, two types of devices are supported:

* modems connected to the User port, these run at 2400 baud
* modems connected via a SwiftLink 232 (or clone) cartridge. These can run at any supported baud rate.

Ethernet cartridges of any type, are currently not supported.
